ld/src/ld/parsers/libunwind/DwarfInstructions.hpp

```diff
@@ -186,6 +187,14 @@ const char* DwarfInstructions<A,R>::parseCFIs(A& addressSpace, pint_t ehSectionS
                                       bool keepDwarfWhichHasCU,  bool forceDwarfConversion, bool neverConvertToCU,
                                       CFI_Atom_Info<A>* infos, uint32_t& infosCount, void* ref, WarnFunc warn)
 {
+    std::unordered_set<int> cuStartsSet;
+    cuStartsSet.reserve(cuCount);
+    for (uint32_t i = 0; i < cuCount; i++) {
+        if (cuStartsSet.find(cuStarts[i]) == cuStartsSet.end()) {
+            cuStartsSet.insert(cuStarts[i]);
```

Comment:
let's replace this find-insert combo with `.emplace` to do 1 hash lookup instead of two